data/fields/fuel/fuel_multi-RU.json

```diff
@@ -2,6 +2,37 @@
     "key": "fuel:",
     "type": "multiCombo",
     "label": "{fuel/fuel_multi}",
+    "strings": {
+        "options": {
+            "diesel": "Diesel",
+            "GTL_diesel": "GTL Diesel",
+            "HGV_diesel": "Truck Diesel",
+            "biodiesel": "Biodiesel",
+            "diesel:class2": "Cold-weather Diesel",
+            "octane_80": "Gasoline (80 Octane)",
+            "octane_85": "Gasoline (85 Octane)",
+            "octane_87": "Gasoline (87 Octane)",
+            "octane_88": "Gasoline (88 Octane)",
+            "octane_89": "Gasoline (89 Octane)",
+            "octane_90": "Gasoline (90 Octane)",
+            "octane_91": "Gasoline (91 Octane)",
+            "octane_92": "Gasoline (92 Octane)",
+            "octane_93": "Gasoline (93 Octane)",
+            "octane_95": "Gasoline (95 Octane)",
+            "octane_97": "Gasoline (97 Octane)",
+            "octane_98": "Gasoline (98 Octane)",
+            "octane_100": "Gasoline (100 Octane)",
+            "e10": "E10 Gasoline",
+            "e85": "E85 Gasoline",
+            "lpg": "Liquefied Natural Gas (LPG)",
+            "cng": "Compressed Natural Gas (CNG)",
+            "propane": "Propane",
+            "LH2": "Liquid Hydrogen",
+            "h70": "Gaseous Hydrogen (700 bar)",
+            "adblue": "AdBlue / AUS 32",
+            "adblue:canister": "AdBlue / AUS 32 (sold in canisters)"
```

Comment:
@tyrasd, do you know if it’s necessary to copy the whole object here? Does `stringsCrossReference` merge the cross-referenced object with the object defined here? Copying all the strings here would create some extra work for translators – not too much right now, but it could become more pronounced as we add more country-specific versions of this field.

Reply:
Well, `stringsCrossReference` indicates that the client should look up the whole `strings` property of the referenced field. Currently it's undefined what should happen when one defines both `stringsCrossReference` and `strings` in a field (FWIW, iD would currently throw away the locally defined strings).
The currently intended way for this to be used is that the global field defines all strings, and the regional variants of the field then only restrict the allowed options by defining their own `options` property.[^1]

[^1] In the case where (some) options do not fit for the global case, the strings can still be defined there, but the global field would also need to define its own `options` list.